README.md

```diff
@@ -106,7 +106,7 @@ We maintain a Docker image preconfigured with the Azure CLI.
 See our [Docker tags](https://hub.docker.com/r/microsoft/azure-cli/tags/) for available versions.
 
 ```bash
-$ docker run -v ${HOME}:/root -it --rm mcr.microsoft.com/azure-cli:<version>
+$ docker run -u $(id -u):$(id -g) -v ${HOME}:/home/az -e HOME=/home/az --rm -it mcr.microsoft.com/azure-cli:<version>
```

Comment:
I suggest we keep it as it is unless we find the option which would work cross OSs including Windows. 
Thanks for sharing the doc though where I learnt some good things.

Reply:
Hi @yugangw-msft no problem. But since the initial command works only in bash (${HOME} variable) this suggestion will work in a bash on windows as well ;-)

Reply:
@kitingChris shall we change [line 115](https://github.com/Azure/azure-cli/pull/10107/files/eacf4399c2d706b46a98ff1a7cd7f24f889757f6#diff-04c6e90faac2675aa89e2176d2eec7d8R115) as well, to keep consistency?

Reply:
@fengzhou-msft yes! Applies there as well. Everywhere where you mount your home directory and you can expect files to be created.